### ISSUE TITLE
Plugin not working with bazel 0.26.0 (missing input file '@remote_java_tools_darwin//:Genclass')

### ISSUE THREAD

**Issue by natros** [bazelbuild/intellij#845]
Works from command link but not with intellij plugin. I have no problems with bazel 0.25.3 

```
Syncing project: Sync (incremental)...
Updating VCS...
Running Bazel info...
Command: /usr/local/bin/bazel info --tool_tag=ijwb:IDEA:ultimate --curses=no --color=yes --progress_in_terminal_title=no --

Command: git diff --name-status --no-renames b5e7a496c71df57dbd277deaff615d0589ad3eaa

Computing VCS working set...
  .bazelproject (modified)
  .vscode/settings.json (added)
  candidaturas/BUILD.bazel (added)
  candidaturas/src/main/resources/pt/ipb/foo/conf.xml (added)
  docs/java/pt/ipb/docs/dto/UserDto.java (modified)

Sync targets from project view:
  //...:all

Sync targets from working set:
  //:all
  //candidaturas:all
  //docs/java/pt/ipb/docs/dto:all

Building blaze targets...
Command: /usr/local/bin/bazel build --tool_tag=ijwb:IDEA:ultimate --keep_going --build_event_binary_file=/var/folders/gk/1s9chv9j0291fhtc2dkpr6hh0000gn/T/intellij-bep-90bde706-21b7-4232-afce-d775d00c05fa --nobuild_event_binary_file_path_conversion --curses=no --color=yes --progress_in_terminal_title=no --aspects=@intellij_aspect//:intellij_info_bundled.bzl%intellij_info_aspect --override_repository=intellij_aspect=/Users/fsousa/Library/Application Support/IntelliJIdea2019.1/ijwb/aspect --output_groups=intellij-info-generic,intellij-info-java,intellij-info-py,intellij-resolve-java,intellij-resolve-py -- //...:all //:all //candidaturas:all //docs/java/pt/ipb/docs/dto:all

Loading: 
Loading: 0 packages loaded
Analyzing: 97 targets (0 packages loaded, 0 targets configured)
Analyzing: 97 targets (0 packages loaded, 0 targets configured)
INFO: Analyzed 97 targets (0 packages loaded, 0 targets configured).
INFO: Found 97 targets...
[2 / 6] [Prepa] BazelWorkspaceStatusAction stable-status.txt
ERROR: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/demo/java/pt/ipb/demo/dagger/example5/BUILD.bazel:1:1: //demo/java/pt/ipb/demo/dagger/example5:example5: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/wf/java/com/gitlab/natros/wf/guice/lifecycle/BUILD.bazel:1:1: //wf/java/com/gitlab/natros/wf/guice/lifecycle:lifecycle: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/docs/java/pt/ipb/docs/api/BUILD.bazel:1:1: //docs/java/pt/ipb/docs/api:api: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/demo/java/pt/ipb/demo/dagger/coffe/shopper/BUILD.bazel:1:1: //demo/java/pt/ipb/demo/dagger/coffe/shopper:shopper: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/demo/java/pt/ipb/demo/dagger/example2/BUILD.bazel:1:1: //demo/java/pt/ipb/demo/dagger/example2:example2: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/wf/java/com/gitlab/natros/wf/gwt/restygwt/BUILD.bazel:1:1: //wf/java/com/gitlab/natros/wf/gwt/restygwt:restygwt: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/demo/javatests/pt/ipb/demo/dagger/instance/BUILD.bazel:1:1: //demo/javatests/pt/ipb/demo/dagger/instance:BindInstanceTest: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/docs/java/pt/ipb/docs/ui/login/BUILD.bazel:1:1: //docs/java/pt/ipb/docs/ui/login:login: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/docs/java/pt/ipb/docs/ui/welcome/BUILD.bazel:1:1: //docs/java/pt/ipb/docs/ui/welcome:welcome: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/demo/java/pt/ipb/demo/dagger/example4/BUILD.bazel:1:1: //demo/java/pt/ipb/demo/dagger/example4:example4: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/wf/java/com/gitlab/natros/wf/gwt/core/BUILD.bazel:1:1: //wf/java/com/gitlab/natros/wf/gwt/core:core: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/demo/java/pt/ipb/demo/dagger/example3/BUILD.bazel:1:1: //demo/java/pt/ipb/demo/dagger/example3:example3: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/docs/java/pt/ipb/docs/api/gwt/BUILD.bazel:1:1: //docs/java/pt/ipb/docs/api/gwt:gwt: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/docs/java/pt/ipb/docs/ui/app/BUILD.bazel:1:1: //docs/java/pt/ipb/docs/ui/app:app: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/docs/java/pt/ipb/docs/server/user/BUILD.bazel:1:1: //docs/java/pt/ipb/docs/server/user:user: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/ipb/monorepo/demo/java/pt/ipb/demo/factory/BUILD.bazel:1:1: //demo/java/pt/ipb/demo/factory:factory: missing input file '@remote_java_tools_darwin//:Genclass'
INFO: Elapsed time: 1.750s, Critical Path: 0.00s
INFO: 0 processes.
FAILED: Build did NOT complete successfully
INFO: Build Event Protocol files produced successfully.
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully
build invocation result: build_error
Sync was successful, but there were Bazel build errors. The project may not be fully updated or resolve until fixed. If the errors are from your working set, please uncheck 'Bazel > Sync > Expand Sync to Working Set' and try again.
```

```
$ bazel info
bazel-bin: /private/var/tmp/_bazel_fsousa/187058d15007c884f8885295a818c294/execroot/monorepo/bazel-out/darwin-fastbuild/bin
bazel-genfiles: /private/var/tmp/_bazel_fsousa/187058d15007c884f8885295a818c294/execroot/monorepo/bazel-out/darwin-fastbuild/bin
bazel-testlogs: /private/var/tmp/_bazel_fsousa/187058d15007c884f8885295a818c294/execroot/monorepo/bazel-out/darwin-fastbuild/testlogs
character-encoding: file.encoding = ISO-8859-1, defaultCharset = ISO-8859-1
command_log: /private/var/tmp/_bazel_fsousa/187058d15007c884f8885295a818c294/command.log
committed-heap-size: 578MB
execution_root: /private/var/tmp/_bazel_fsousa/187058d15007c884f8885295a818c294/execroot/monorepo
gc-count: 63
gc-time: 4460ms
install_base: /var/tmp/_bazel_fsousa/install/37d3a264d720ec9ae63603a543c8ff7c
java-home: /private/var/tmp/_bazel_fsousa/install/37d3a264d720ec9ae63603a543c8ff7c/_embedded_binaries/embedded_tools/jdk
java-runtime: OpenJDK Runtime Environment (build 11.0.2+7-LTS) by Azul Systems, Inc.
java-vm: OpenJDK 64-Bit Server VM (build 11.0.2+7-LTS, mixed mode) by Azul Systems, Inc.
max-heap-size: 3817MB
output_base: /private/var/tmp/_bazel_fsousa/187058d15007c884f8885295a818c294
output_path: /private/var/tmp/_bazel_fsousa/187058d15007c884f8885295a818c294/execroot/monorepo/bazel-out
package_path: %workspace%
release: release 0.26.0
repository_cache: /var/tmp/_bazel_fsousa/cache/repos/v1
server_log: /private/var/tmp/_bazel_fsousa/187058d15007c884f8885295a818c294/java.log.caronte.fsousa.log.java.20190528-123401.45561
server_pid: 45561
used-heap-size: 209MB
workspace: /Users/fsousa/Projects/ipb/monorepo
```

```
IntelliJ IDEA 2019.1.3 (Ultimate Edition)
Build #IU-191.7479.19, built on May 28, 2019
JRE: 11.0.2+9-b159.60 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.5
```

**Comment by jin** [comment]
I cannot reproduce this on macOS with Bazel 0.26.0. This missing file belongs to an external repository that may have been updated between 0.25 and 0.26. It's odd that the external repository wasn't invalidated -- could you please try `bazel clean --expunge` and try again?

**Comment by natros** [comment]
done that and the problem remains.

could you please teste this https://github.com/natros/bazel-maven:
`git clone git@github.com:natros/bazel-maven.git`
 and see if you can reproduce the error. (it only happens inside intellij, works perfectly well from comand line)

```
...
[9 / 16] [Prepa] BazelWorkspaceStatusAction stable-status.txt
ERROR: missing input file '@remote_java_tools_darwin//:Genclass'
ERROR: /Users/fsousa/Projects/tmp/bazel-maven/java/com/gitlab/natros/test/BUILD.bazel:1:1: //java/com/gitlab/natros/test:foo: missing input file '@remote_java_tools_darwin//:Genclass'
....
```

**Comment by jin** [comment]
Thanks for the repro! I can reproduce the error with this:

```
$ bazel build   --aspects=@intellij_aspect//:intellij_info_bundled.bzl%intellij_info_aspect \
  --override_repository=intellij_aspect="$HOME/Library/Application Support/IdeaIC2019.1/ijwb/aspect" \
  --output_groups=intellij-resolve-java -- //...:all
```

If you omit the `intellij-resolve-java` output group, it works fine.

**Comment by jin** [comment]
Found the root cause: https://github.com/bazelbuild/bazel/pull/8490

**Comment by jin** [comment]
This fix will be part of a patch release for 0.26

**Comment by jin** [comment]
Reopening until the patch fix for Bazel is out

**Comment by jin** [comment]
More updates here https://github.com/bazelbuild/java_tools/issues/8

**Comment by jin** [comment]
This is now fixed with 0.26.1.
